Neos.Fusion/Classes/Core/Runtime.php

```diff
@@ -217,16 +217,14 @@ public function injectSettings(array $settings)
      *
      * During Fusion rendering the method can be used to add tag dynamicaly for the current cache segment.
      *
-     * @param string $key
-     * @param string $value
-     * @return void
+     * @throws Exceptions
      */
-    public function addCacheTag($key, $value)
+    public function addCacheTag(string $tag): void
```

Comment:
lol am i dumb or shouldnt this be the new dto here??????????

but my brain is dead and im about to fall asleep in the train

note to myself: dont forget my suitcase above me in the bahn